src/services/control-panel-service.ts

```diff
@@ -514,7 +514,7 @@ export class ControlPanelService implements ControlPanelService {
             targetBlank: true,
           },
           {
-            linkUrl: `control_panel.external_integrations.dynamics_link_url`,
+            linkUrl: _('control_panel.external_integrations.dynamics_link_url'),
             imgSrc: dynamics_icon,
             imgAlt: 'control_panel.external_integrations.dynamics_title',
             iconName: 'control_panel.external_integrations.dynamics_title',
```

Comment:
These fields are also translatable, what do you think about translating all translatable things in `getControlPanelSections`?

Reply:
```suggestion
            imgAlt: _('control_panel.external_integrations.dynamics_title'),
            iconName: _('control_panel.external_integrations.dynamics_title'),
```